src/jaxsim/api/data.py

```diff
@@ -417,7 +417,7 @@ def base_velocity(self) -> jtp.Vector:
     def generalized_position(self) -> tuple[jtp.Matrix, jtp.Vector]:
         """
         Get the generalized position
-        :math:`\mathbf{q} = ({}^W \mathbf{H}_B, \mathbf{s}) \in \text{SO}(3) \times \mathbb{R}^n`.
+        :math:`\\mathbf{q} = ({}^W \\mathbf{H}_B, \\mathbf{s}) \\in \text{SO}(3) \times \\mathbb{R}^n`.
```

Comment:
Can this be fixed using `r"""` as in other docstrings?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/114/commits/f7c63e4ff1a69bb6efd5d40b15544994c938e2ee